### PR TITLE
Add support for libstaticassets

### PR DESCRIPTION
**Pull request by pbiggar**
Add StaticAssets module. I opted not to use the legacy http clients, but to replicate what we used them for instead.

Also add Twilio module - I used the legacy clients for that.

### CODE REVIEW

**Comment by pbiggar** [comment]
predeployment check error can be ignored

File: fsharp-backend/src/BackendOnlyStdLib/LibTwilio.fs [hunk 1]
```diff
@@ -15,125 +17,102 @@ let incorrectArgs = LibExecution.Errors.incorrectArgs
 let varA = TVariable "a"
 let varB = TVariable "b"
```
Comment (StachuDotNet) [inline fsharp-backend/src/BackendOnlyStdLib/LibTwilio.fs:18]:
```suggestion
```
unused binding

File: fsharp-backend/src/BackendOnlyStdLib/LibTwilio.fs [hunk 2]
```diff
@@ -15,125 +17,102 @@ let incorrectArgs = LibExecution.Errors.incorrectArgs
...
+            $"https://api.twilio.com/2010-04-01/Accounts/{accountSID}/Messages.json"
+          LegacyHttpClient0.sendRequest
+            twilioUri
+            System.Net.Http.HttpMethod.Post
+            LibExecution.DvalReprExternal.toPrettyMachineJsonStringV1
+            (Some body)
+            (DObj Map.empty)
+            headers
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplementedTODO
```
Comment (StachuDotNet) [inline fsharp-backend/src/BackendOnlyStdLib/LibTwilio.fs:67]:
```suggestion
      sqlSpec = NotQueryable
```

File: fsharp-backend/src/BackendOnlyStdLib/LibTwilio.fs [hunk 3]
```diff
@@ -15,125 +17,102 @@ let incorrectArgs = LibExecution.Errors.incorrectArgs
...
```
Comment (StachuDotNet) [inline fsharp-backend/src/BackendOnlyStdLib/LibTwilio.fs:116]:
```suggestion
      sqlSpec = NotQueryable
```

**Review by StachuDotNet — CHANGES_REQUESTED** [review]
Minor suggestions. There are also a few errant "in"s hanging around in LibTwilio.fs

**Pull request merged**